src/epivizFileParser/SplicingBamFile.py

```diff
@@ -0,0 +1,136 @@
+import pysam
+import numpy
+from .BamFile import BamFile
+from .utils import toDataFrame
+
+__author__ = "Jayaram Kancherla, Victor Alexandru"
+__copyright__ = "jkanche"
+__license__ = "mit"
+
+
+class SplicingBamFile(BamFile):
+    """
+    Class to parse bam files 
+
+    Args:
+        file (str): file location can be local (full path) or hosted publicly
+        columns ([str]) : names of columns in the bam file
+
+    Attributes:
+        file: a pysam file object
+        fileSrc: location of the file
+        cacheData: cache of accessed data in memory
+        columns: column names
+    """
+
+    def __init__(self, file, columns=None):
+        super().__init__(file, columns)
+
+    def get_col_names(self):
+        """
+        Columns of a bam file
+        """
+        if self.columns is None:
+            self.columns = ["chr", "region1_start", "region2_start",
+                            "region1_end", "region2_end", "value"]
+        return self.columns
+
+    def _get_junctions(self, chr, start, end):
+        iter = self.file.fetch(chr, start, end)
+
+        wiggle = numpy.zeros(end - start + 1)
```

Comment:
Looks like `wiggle` is not used (not being returned), suggest that everything having to do with this is removed

Reply:
I do not know the meaning behind wiggle (a bioinformatics concept?). I was advised to leabe it in like it is in the source code.

Reply:
You can remove it